package.json

```diff
@@ -20,58 +21,22 @@
     "url": "https://github.com/papercss/papercss/issues"
   },
   "homepage": "https://www.getpapercss.com",
-  "dependencies": {
-    "gulp": "^4.0.0",
-    "gulp-autoprefixer": "^6.0.0",
-    "gulp-clean-css": "3.9.0",
-    "gulp-connect": "^5.6.1",
-    "gulp-plumber": "1.1.0",
-    "gulp-rename": "1.2.2",
-    "gulp-sass": "^3.1.0",
-    "optional": "^0.1.4"
-  },
+  "dependencies": {},
   "devDependencies": {
-    "browserify": "^14.1.0",
+    "autoprefixer": "^9.8.6",
     "chai": "^4.1.2",
-    "gulp-less": "^4.0.1",
-    "gulp-plumber": "1.1.0",
-    "gulp-rename": "1.2.2",
-    "gulp-sass": "^3.1.0",
-    "gulp-stylelint": "6.0.0",
-    "gulp-watch-less": "1.0.1",
-    "http-server": "^0.11.1",
-    "hugo-bin": "^0.37.0",
```

Comment:
I think we'll want to keep `hugo-bin` for local development.

Reply:
I've re-added hugo-bin as a development dependency. However, I don't really know why this is required as we can directly run it with `hugo --source=docs ...`.

Maybe it is for Windows users?

Reply:
Thank you! It's mostly for those who don't have `hugo` installed globally, they can still run `hugo` commands within `npm` scripts.